dozer/cogs/levels.py

```diff
@@ -252,9 +248,31 @@ def _ensure_sync_running(self):
         finally:
             self.sync_task.start()
 
-    @command()
+    @command(aliases=["rolelevels", "levelroles"])
     @guild_only()
-    @has_permissions(administrator=True)
+    async def checkrolelevels(self, ctx):
+        """Displays all level associated roles"""
+        roles = sorted(self._level_roles.get(ctx.guild.id), key=lambda entry: entry.level)  # Sort roles based on level
+        e = discord.Embed(title=f"Level roles for {ctx.guild}", color=blurple)
+        if roles:
+            e.description = f"This server has {len(roles)} level roles"
+            for level_role in roles:
+                role = ctx.guild.get_role(level_role.role_id)
+                if_unavailable = f"<@{level_role.role_id}>"
```

Comment:
If the role doesn't exit, its mention isn't easy for a user to understand. This message should be that the role has been deleted or something similar.